src/model/censusdata/services/geodataApiDataService.js

```diff
@@ -0,0 +1,54 @@
+import { csvParse } from "d3-dsv";
+import { writeDataToMapObj } from "../../utils";
+
+const baseURL = "https://5laefo1cxd.execute-api.eu-central-1.amazonaws.com/dev/hello/skinny";
+
+export default class GeodataApiDataService {
+  async fetchAllDataForGeography(geographyId) {
+    const url = `${baseURL}?rows=${geographyId}`;
+    const response = await fetch(url);
+    const string = await response.text();
+    let data = new Map();
```

Comment:
you've got the same data key as `geographyId` here but `geography_code` elsewhere (unless I've misread) - probably better to be consistent?

Reply:
yep good point and a bit of an issue across the repo

Reply:
👍